ui/values/localizable/es.go

```diff
@@ -90,7 +90,8 @@ const ES = `
 "viewProperty" = "Ver propiedad";
 "stakeShuffle" = "StakeShuffle";
 "rename" = "Renombrar";
-"renameWalletSheetTitle" = "Renombrar la cartera";
+"renameWalletSheetTitle" = "Change wallet name";
+"securityToolsInfo" = "%v Various tools that help in different aspects of crypto currency security will be located here. %v"
```

Comment:
This should be translated to spanish

Reply:
The idea is to have a place holder for the other languages pending when we do a general sting localization like the https://github.com/planetdecred/godcr/pull/906